### ISSUE TITLE
Inserting item via CollectionView.insertItems gives Jerk effect

### ISSUE THREAD

**Issue by SureshSc** [dejanskledar/centered-collectionview-layout#3]

**Comment by dejanskledar** [comment]
Yes I noticed this. I presume implementing the missing required methods would fix the issue, but I didn't have the time to test this. By the docs:

Every layout object should implement the following methods:
- collectionViewContentSize
- layoutAttributesForElements(in:)
- layoutAttributesForItem(at:)
- layoutAttributesForSupplementaryView(ofKind:at:) (if your layout supports supplementary views)
- layoutAttributesForDecorationView(ofKind:at:) (if your layout supports decoration views)
- shouldInvalidateLayout(forBoundsChange:)
